### DISCUSSION TITLE
Team development with different branches.

### DISCUSSION THREAD

**Discussion by AntoineGGG** [prisma/prisma#10502] [Q&A] [answered]
Hi guys, 
First thanks for you work, I use prisma for 5 month now and really enjoy it ! 
One question remain when we work with my colleagues:

I follow this workflow:  https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate/team-development/
I understand it but:

Let's imagine if Ania, during her work, remove the "name" field on user model, and change it with "firstname".
At the same time I changed on my own branch the type of the name field.
We push our work on the dev branch, but the migration will crash because if Ania change the name field before I change the type, prisma will say that the field name doesn't exist on user model....
Is there a solution to avoid this problem ? 

Best Regards 
Antoine

**Comment by TasinIshmam** [comment, marked as answer]
Hey @AntoineGGG :wave:  

Unfortunately, I don't have any concrete suggestions about this aside from being very mindful and coordinating any database schema changes across the team (to avoid a situation like the one you described). 

As an additional note (though not currently a solution to the problem you mentioned), better team collaboration is something we hope to improve with the [Prisma Data Platform](https://cloud.prisma.io/), which is currently in early access. Right now, it supports viewing/editing your Prisma Schema online and in collaboration with team members. Perhaps you'd like to check it out!

**Reply by AntoineGGG** [reply]
Hi @TasinIshmam,
Thanks for your reply, I'll have a look at Prisma Data Platform!!

Regards
Antoine

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
